src/application/service/ExchangeAuthCode.test.ts

```diff
@@ -64,10 +68,10 @@ describe('ExchangeAuthCode', () => {
   it('creates a new token for a valid link', async () => {
     const email = new Email('kostas@tw.ee');
     const link = await magicLinkRepository.save(
-      new MagicLink(v4(), email, v4())
+      new MagicLink(new MagicLinkCode(new MagicLinkCode().value), email)
```

Comment:
Shouldn't this be 
new MagicLink(new MagicLinkCode().value, email)

Reply:
Yeah, looks like you have it twice.

Reply:
whooops. yeah.. this looks like a bit  too eager 😅 Good catch